GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+Four weeks or more before the election, the Election Officer(s) will issue a call for nominations, publish the list of voters. 
+
+Seven days before the election, the call for nominations will be closed. The election will be open for voting for no less than two weeks and no more than four. The results of the election will be announced within one week of closing the election. New Steering Committee members will take office is May or June of each year.
+
+### Election Officers
+
+Six weeks or more before the election, the Steering Committee will appoint three Election Officer(s) to administer the election. Election Officers will be Organization Members in good standing who are eligible to vote, are not running for Steering Committee in that election, who are not currently part of the Steering Committee and can make a public promise of impartiality. 
+
+They will be responsible for:
+
+* Making all announcements associated with the election
+* Preparing and distributing electronic ballots
+* Assisting candidates in preparing and sharing statements
+* Tallying voting results according to the rules in this governance
+
+### Eligibility to Vote
+
+Anyone who is an Organization Member as defined in [ROLES.md](#roles.md).
+
+The electoral roll of all eligible voters will be captured at [elections](./elections/) and the voters’ guide will be captured at [voters guide](./elections/).
+
+### Candidate Eligibility
+
+Community members must be eligible to vote to stand for election. Candidates should self-nominate. There are no term limits for Steering Committee members.
+
+### Voting Procedure
+
+Elections will be held using a ranked choice voting system which supports voter privacy.  Specific voter tooling will be selected by the Election Officers before the election starts, with this selection to be approved by the Steering Committee. The top vote-getters will be elected to the open seats, with the exceptions for company representation discussed below.
+
+### Limitations on Company Representation
+
+No more than 2 seats may be held by employees of the same organization (or  
+conglomerate, in the case of companies owning each other). If an election results in greater than 2 employees of the same organization being selected, the lowest vote getters from any particular employer will be removed until representation on the committee is down to 2.
+
+If employers change because of job changes, acquisitions, or other events, in a way that would yield too many seats being held by employees of the same organization, and the next regular election is more than 10 weeks away, sufficient members of the committee must resign until only 2 employees of the same employer are left. If it is impossible to find sufficient members to resign, all employees of that organization will be removed and new special elections held. 
```

Comment:
```suggestion
If affiliations change because of job changes, acquisitions, or other events and would result in more than 2 affiliated individuals serving on the Steering Committee and the next regular election is more than 10 weeks away, a sufficient number of members of the Steering Committee must resign. If this does not happen within two weeks, all affiliated individuals from such a set are removed and backfilled through normal backfill mechanisms.
```